src/legacy/ui/public/agg_types/metrics/lib/sibling_pipeline_agg_controller.js

```diff
@@ -1,49 +0,0 @@
-/*
- * Licensed to Elasticsearch B.V. under one or more contributor
- * license agreements. See the NOTICE file distributed with
- * this work for additional information regarding copyright
- * ownership. Elasticsearch B.V. licenses this file to you under
- * the Apache License, Version 2.0 (the "License"); you may
- * not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-
-import { safeMakeLabel } from './safe_make_label';
-import { i18n } from '@kbn/i18n';
-
-const siblingPipelineAggController = function (type) {
-  return function ($scope) {
```

Comment:
This implementation was moved to `SubMetricParamEditor` control
https://github.com/elastic/kibana/pull/39502/files?file-filters%5B%5D=.tsx#diff-309ea246f4f3ee4ba16f9317ec330e8dR37